.github/workflows/pythonapp.yml

```
@@ -35,3 +35,46 @@ jobs:
       run: |
         pip install pytest
         pytest
+    - name: Bump version number
+      if: github.ref == 'refs/heads/master'
+      run: |
+        python -m pip install black
+        echo "NEW_VERSION=$(python .github/workflows/bumper.py)" >> $GITHUB_ENV
+    - name: Commit files
+      if: github.ref == 'refs/heads/master'
+      run: |
+        git config --local user.email "fbda@nfi.nl"
+        git config --local user.name "Version bumper"
+        git add setup.py
+        git commit -m "Bump version to ${{ env.NEW_VERSION }}" || echo "Nothing changed in setup.py"
```

will it ever print the nothing changed?

---

Not in the current configuration I guess, so this would be a better error text.

```suggestion
        git commit -m "Bump version to ${{ env.NEW_VERSION }}" || echo "Committing the version bump failed"
```